src/fuse/mount.rs

```diff
@@ -475,24 +519,45 @@ fn direct_mount(mount_point: &Path, options: &[&str]) -> RawFd {
     }
 
     let mntpath = CString::new(cstr_path).expect("CString::new failed");
-    let fstype = CString::new("fuse").expect("CString::new failed");
-    let fsname = CString::new("/dev/fuse").expect("CString::new failed");
+    let fsname = if let Some(s) = args.get_fsname() {
+        CString::new(&s[..]).expect("CString::new failed")
+    } else if let Some(s) = args.get_subtype() {
+        CString::new(&s[..]).expect("CString::new failed")
```

Comment:
Merge this two?

Reply:
The logic is 
if fsname is set use fsname 
if not use subtype

Reply:
I see. Resolved.